website/docs/components/data-connectors/oracle.md

```diff
@@ -0,0 +1,201 @@
+---
+title: 'Oracle Data Connector'
+sidebar_label: 'Oracle Data Connector'
+description: 'Oracle Data Connector Documentation'
```

Comment:
Needs an order index.

Reply:
It seems the data connector documentation does not use indexes and is automatically ordered alphabetically. Propose keeping the Oracle documentation consistent with this existing approach.